package-lock.json

```diff
@@ -2941,6 +2941,11 @@
 				"uuid": "^3.2.1"
 			},
 			"dependencies": {
+				"adm-zip": {
```

Comment:
as you can see here, there is some diff's on your file.
Please, revert it.

Reply:
i have copied from the rocket chat develop and replace with the my code